lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php

```diff
@@ -438,4 +438,12 @@ public function getPrimaryAlias()
     {
         return $this->primaryAlias;
     }
+
+    /**
+     * Deep clone on converter, so it resets it's constraints.
```

Comment:
This doesn't make much sense to me. Would `Ensure cloned query builder onjects have different instances of the converter` make more sense?